src/fetch_fallback_html.ts

```diff
@@ -11,6 +11,21 @@ export function fetchFallbackHtml (url: string) {
   // eslint-disable-next-line no-empty
   // tslint:disable-next-line: no-empty
   } catch {}
+
+  const { controller, signal } = initAbortController()
   
-  return fetch(url, { headers })
+  return {
+    abort: () => controller && controller.abort(),
+    ready: fetch(url, { headers, ...(signal ? { signal } : {}) })
```

Comment:
Can we instead do: `ready: () => fetch(url... `
and then use it as `request.ready().then(... `

this way abort and ready have the same signatures.

Reply:
Yes, almost even did it that way 👍